src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_diffedit.py

```diff
@@ -1037,7 +1037,7 @@ def generate_mask(
         )
 
         # 4. Preprocess image
-        image = preprocess(image).repeat_interleave(num_maps_per_mask, dim=0)
+        image = self.image_processor.preprocess(image).repeat_interleave(num_maps_per_mask, dim=0)
```

Comment:
Great reuse!

Should we then maybe just delete `preprocess()` to help reduce the LOC from the pipeline source?